src/main/lib.rs

```diff
@@ -2,6 +2,9 @@
  * The Shadow Simulator
  * See LICENSE for licensing information
  */
+#[macro_use]
+extern crate lazy_static;
```

Comment:
Really small thing, but I think the [recommended](https://doc.rust-lang.org/edition-guide/rust-2018/macros/macro-changes.html) way is now to just `use lazy_static::lazy_static;`, and they plan to deprecate (but not remove) `#[macro_use]` at some point.

Reply:
Ah good to know :)